Remora.Discord.Commands/Feedback/Services/IFeedbackService.cs

```diff
@@ -0,0 +1,128 @@
+    Task<Result<IReadOnlyList<IMessage>>> SendPrivateContentAsync(
+        Snowflake user,
+        string contents,
+        Color color,
+        FeedbackMessageOptions? options = null,
+        CancellationToken ct = default
+    );
+
+    /// <summary>
+    ///     Sends a message to the specified channel with optional content, embeds, and custom options.
+    /// </summary>
+    /// <param name="channel">The target channel to send the message.</param>
+    /// <param name="content">Optional content of the message.</param>
+    /// <param name="embeds">Optional list of embeds to include in the message.</param>
+    /// <param name="options">Optional custom message options.</param>
+    /// <param name="ct">Optional CancellationToken to cancel the operation.</param>
+    /// <returns>A Result containing the sent message, or an error if the operation failed.</returns>
+    Task<Result<IMessage>> SendAsync(
+        Snowflake channel,
+        Optional<string> content = default,
+        Optional<IReadOnlyList<IEmbed>> embeds = default,
+        FeedbackMessageOptions? options = null,
+        CancellationToken ct = default
+    );
+
+    /// <summary>
+    ///     Sends a contextual message with optional content, embeds, and custom options.
+    /// </summary>
+    /// <param name="content">Optional content of the message.</param>
+    /// <param name="embeds">Optional list of embeds to include in the message.</param>
+    /// <param name="options">Optional custom message options.</param>
+    /// <param name="ct">Optional CancellationToken to cancel the operation.</param>
+    /// <returns>A Result containing the sent message, or an error if the operation failed.</returns>
+    Task<Result<IMessage>> SendContextualAsync(
+        Optional<string> content = default,
+        Optional<IReadOnlyList<IEmbed>> embeds = default,
+        FeedbackMessageOptions? options = null,
+        CancellationToken ct = default
+    );
+
+    /// <summary>
+    ///     Sends a private message to the specified user with optional content, embeds, and custom options.
+    /// </summary>
+    /// <param name="user">The target user to receive the private message.</param>
+    /// <param name="content">Optional content of the message.</param>
+    /// <param name="embeds">Optional list of embeds to include in the message.</param>
+    /// <param name="options">Optional custom message options.</param>
+    /// <param name="ct">Optional CancellationToken to cancel the operation.</param>
+    /// <returns>A Result containing the sent message, or an error if the operation failed.</returns>
+    Task<Result<IMessage>> SendPrivateAsync(
+        Snowflake user,
+        Optional<string> content = default,
+        Optional<IReadOnlyList<IEmbed>> embeds = default,
+        FeedbackMessageOptions? options = null,
+        CancellationToken ct = default
+    );
+
+    /// <summary>
+    ///     Checks if the current context is an InteractionContext.
```

Comment:
Reuse the existing documentation on FeedbackService where possible. This goes for all the methods.